### PR TITLE
Remove polyfill and update packages

### PR DESCRIPTION
**Pull request by BeeMargarida**
| - | - |
| --- | --- |
| Issue | Derived from https://github.com/ripe-tech/ripe-sdk-components-vue/pull/27 |
| Dependencies | -- |
| Decisions | Remove polyfill and updated packages. |
| Animated GIF | -- |

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -58,15 +58,14 @@
...
+        "ripe-sdk": "^1.12.1",
+        "vue": "^2.6.12",
+        "vue-global-events": "^1.2.1",
+        "yonius": "^0.5.4"
     },
     "devDependencies": {
         "@babel/core": "^7.11.1",
-        "@babel/plugin-transform-runtime": "^7.11.0",
-        "@babel/polyfill": "^7.10.4",
+        "@babel/plugin-syntax-dynamic-import": "^7.8.3",
```
Comment (gcandal) [inline package.json]: Why is this needed?

Comment (BeeMargarida) [reply package.json]: It isn't, will remove

**Review by gcandal — APPROVED** [review]

**Review by joamag — APPROVED** [review]
